readme.md

```diff
@@ -66,8 +76,12 @@ here's a workaround, add this to your pom.xml:
     </dependency>
 ```
 
-## Change History
+## Future Plans
+- interface based java transformer for custom types
+- get as java.util.Properties convenience
```

Comment:
Isn't it what you just implemented in this PR ?

Reply:
yes it is :)